README.md

```diff
@@ -1,41 +1,61 @@
-
 mapstory
 ================
 
 [![Coverage Status](https://coveralls.io/repos/github/MapStory/mapstory/badge.svg?branch=master)](https://coveralls.io/github/MapStory/mapstory?branch=master) [![Build Status](https://travis-ci.org/MapStory/mapstory.svg?branch=master)](https://travis-ci.org/MapStory/mapstory)
 
-Please visit [our wiki](https://github.com/MapStory/mapstory/wiki)
+MapStory.org is the free atlas of change that everyone can edit. We are a community, not a company, working to organize humanity’s shared knowledge about how the world evolves geographically over time, and to make this knowledge easily accessible as an open educational resource.
+
+The MapStory community’s work breaks down into 3 big tasks. We call them the “3 Cs”. First, once you’ve created a profile, you can _collect_ spatio-temporal data and import it. We call each dataset imported into MapStory.org a “StoryLayer”. Second you can _compose_ MapStories that provide nuanced understanding of historical change by combining StoryLayers and other narrative elements, like images, text, or video. Finally, you can _curate_ the accuracy and quality of content presented in MapStory.org by adding ratings, checking metadata, making comments and committing version edits to the actual StoryLayers themselves, much like you might edit a Wikipedia page.
+
+
+## Getting Started
+
+
+### Prerequisites
+
+This guide assumes you are using a UNIX based operating system such as Linux or OSX and have the below installed.
 
-Interested in seeing our progress?  We're keeping track of what is going on using our [Waffle kanban board](https://waffle.io/MapStory/mapstory).
+```
+git
+docker
+docker-compose
+```
+### Installation
```

Comment:
Maybe instead of "Installation" we call this "Local Setup", "Setup a Developer Environment", or similar (which matches the description on L25)